content/episode/zee-spencer-cohere-zinc-coop.md

```diff
@@ -0,0 +1,614 @@
+
+{{< timestamp "[00:10:11]" >}}
+And you know, they've made a lot of money and they continue to make a lot of money and I didn't make much after I left because I just couldn't even like exercise the options or anything. I was just too, too busted after that.
+
+{{< timestamp "[00:10:24]" >}}
+_Ana:_ It happens to a lot of people, by the way, I just want to say that story you're telling is super common.
+
+{{< timestamp "[00:10:29]" >}}
+_Zee:_ Yeah. And it's really obvious what the problem is, right? Like it's not just like some incredibly difficult issue. We have expectations that are unreasonable. Like, hey, you want to turn $1 into a hundred dollars, in like 10 years, and you want to do that without doing anything except for putting in a dollar then, like, where is the energy going to come from to turn that $1 into a hundred dollars?
+
+{{< timestamp "[00:10:53]" >}}
+It's either going to come from your customers, by exploiting them and treating them like shit, which is, you know, the Facebook model. Or it's going to come from your employees, which you know, is the easier pathway to take, because if you don't have a high emphasis on customer happiness and customer success, and like actually meeting their needs as a for-profit business, you're not going to succeed.
+
+{{< timestamp "[00:11:11]" >}}
+And so you wind up with this tension between this, you know, 10 X growth every three to four years, and "Hey, we really need to care for our customers. We really need to make them successful." And the people who are caught in the middle of it are your customer support team. They're your ops team, your engineers, your marketing team.
+
+{{< timestamp "[00:11:28]" >}}
+They're all just like "Well, okay. I guess." And because we get paid so well in the tech industry, we just kind of accept it. We just accept that, hey, my health, my happiness, my ability to be present for my significant other, all of those things are less important because I'm making 180, 250, 350 thousand a year.
+
+{{< timestamp "[00:11:48]" >}}
+You can't sell your life like that. I mean, you can try, but at some point it just like breaks you. Or you get really rich and then you think it was all worth it. And you go on the speaking circuit and you like, you know, go on Hacker News, apparently, if that's still a thing. I don't know. I stopped reading the internet a while ago.
+
+{{< timestamp "[00:12:06]" >}}
+_Ana:_ Yeah, the orange site is still a thing. I do not recommend it. So you went back to that startup and you met some of your current business partners. So what happened?
+
+{{< timestamp "[00:12:17]" >}}
+_Zee:_ Yeah, so I went back to my old S-corp and I was like, "All right, I'm going to dust off the old Rolodex", an, an old colleague, Attursh, was like, "Hey, would you like to do some Rails work for Code for America?" And I was like, "that seems like a great way to stick the landing  and do something meaningful."
```

Comment:
@zspencer Can you confirm what is the right spelling of "Attursh"'s name? (I have a strong feeling this has been badly mangled in this transcript...).

Reply:
Fixed.